libmscore/types.h

```diff
@@ -82,7 +82,7 @@ enum class ElementType {
       HARMONY,
       FRET_DIAGRAM,
       BEND,
-      TREMOLOBAR,
+      TREMOLO_BAR,
```

Comment:
`ElementType` enum is exposed to plugins framework so it might break compatibility with some plugins. It might happen that no plugins use it, but maybe we should avoid such changes if no major restructuring of libmscore occurs.

Reply:
You can close this PR if you want, no one will blame you ;-)